<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arisu</class>
 <widget class="QWidget" name="Arisu">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 244, 249);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="TitleBar">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0" rowspan="2">
      <layout class="QHBoxLayout" name="TitleLayout" stretch="0,0,0">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="Logo">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/Logo/Logo/32.ico</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="SoftwareName">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>爱丽丝QQ聊天AI 1.0.0</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="TitleSpace">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="WindowControlButtons">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="hide_btn">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>深度隐藏（去任务管理器找）</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background: transparent;  /* 透明背景 */
    border: none;            /* 移除默认边框 */
}
/* 悬停状态 */
QPushButton:hover {
    background: rgb(233, 233, 233); /* 轻微高亮 */
}
/* 按下状态 */
QPushButton:pressed {
    background: rgb(213, 213, 213);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/标题栏/标题栏/深度隐藏.png</normaloff>:/标题栏/标题栏/深度隐藏.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="min_system_tray_btn">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>隐藏到托盘</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background: transparent;  /* 透明背景 */
    border:  transparent;            /* 移除默认边框 */
}
/* 悬停状态 */
QPushButton:hover {
    background: rgb(233, 233, 233); /* 轻微高亮 */
}
/* 按下状态 */
QPushButton:pressed {
    background: rgb(213, 213, 213);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/标题栏/标题栏/托盘隐藏.png</normaloff>:/标题栏/标题栏/托盘隐藏.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="top_btn">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>窗口置顶</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background: transparent;  /* 透明背景 */
    border:  transparent;            /* 移除默认边框 */
}
/* 悬停状态 */
QPushButton:hover {
    background: rgb(233, 233, 233); /* 轻微高亮 */
}
/* 按下状态 */
QPushButton:pressed {
    background: rgb(213, 213, 213);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/标题栏/标题栏/窗口置顶.png</normaloff>:/标题栏/标题栏/窗口置顶.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="min_btn">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>最小化</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background: transparent;  /* 透明背景 */
    border: none;            /* 移除默认边框 */
}
/* 悬停状态 */
QPushButton:hover {
    background: rgb(233, 233, 233); /* 轻微高亮 */
}
/* 按下状态 */
QPushButton:pressed {
    background: rgb(213, 213, 213);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/标题栏/标题栏/最小化.png</normaloff>:/标题栏/标题栏/最小化.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="max_btn">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>最大化/恢复</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background: transparent;  /* 透明背景 */
    border: none;            /* 移除默认边框 */
}
/* 悬停状态 */
QPushButton:hover {
    background: rgb(233, 233, 233); /* 轻微高亮 */
}
/* 按下状态 */
QPushButton:pressed {
    background: rgb(213, 213, 213);
}
</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/标题栏/标题栏/最大化.png</normaloff>:/标题栏/标题栏/最大化.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_btn">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>关闭</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background: transparent;  /* 透明背景 */
    border:  transparent;            /* 移除默认边框 */
}
/* 悬停状态 */
QPushButton:hover {
    background: rgb(196, 43, 28); /* 轻微高亮 */
}
/* 按下状态 */
QPushButton:pressed {
    background: rgb(199, 64, 49);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/标题栏/标题栏/关闭.png</normaloff>:/标题栏/标题栏/关闭.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 设置选项卡最小/最大尺寸 */
QTabBar::tab {
	border-right:  1px solid rgb(226, 230, 234);  /* 右边框显示颜色 */
	border-top-left-radius: 10px;    /* 左上角圆角半径 */
	border-bottom-right-radius: 0px; /* 右下角直角 */
	min-width: 70px;  /* 最小宽度 */
	min-height: 50px;  /* 最小高度 */
}

/* 悬停状态 */
QTabBar::tab:hover {
    background:rgb(170, 255, 255);
	border-radius: 10px;		/*圆角*/
}

/* 选中状态 */
QTabBar::tab:selected {
	border-top-right-radius: 10px;   /* 右上角圆角半径 */
	border-bottom-left-radius: 0px;  /* 左下角直角 */
	border-bottom-right-radius: 0px; /* 右下角直角 */
	background-color: rgb(232, 235, 240);
    color: white;             /* 选中文字色 */
    border-bottom: 10px solid rgb(85, 255, 255); /* 底部指示条 */
}</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::TabPosition::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::TabShape::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="Home">
      <property name="styleSheet">
       <string notr="true">transform: rotate(0deg);	/* 强制文本水平显示 */</string>
      </property>
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/选项卡图标/选项卡图标/主页.png</normaloff>:/选项卡图标/选项卡图标/主页.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>主页</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="StateMonitor">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/选项卡图标/选项卡图标/状态监控.png</normaloff>:/选项卡图标/选项卡图标/状态监控.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>状态监测</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="widget">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/选项卡图标/选项卡图标/热键.png</normaloff>:/选项卡图标/选项卡图标/热键.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>快捷键</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="QuestionLinks">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/选项卡图标/选项卡图标/问题链接.png</normaloff>:/选项卡图标/选项卡图标/问题链接.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>问题链接</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="QuestionLinksBackground">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	border-radius: 10px;		/*圆角*/
		border: none;			/*无边框*/
}</string>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/背景/背景/满充能.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Settings">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/选项卡图标/选项卡图标/用户设置.png</normaloff>:/选项卡图标/选项卡图标/用户设置.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>用户设置</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
